source/cpu/cpuid_arm64.go

```diff
@@ -78,6 +87,15 @@ var flagNames_arm64 = map[uint64]string{
 	CPU_ARM64_FEATURE_ASIMDDP:  "ASIMDDP",
 	CPU_ARM64_FEATURE_SHA512:   "SHA512",
 	CPU_ARM64_FEATURE_SVE:      "SVE",
+	CPU_ARM64_FEATURE_ASIMDFHM: "ASIMDFHM",
+	CPU_ARM64_FEATURE_DIT:      "DIT",
```

Comment:
@yselkowitz Where is the list of arm cpu features coming from? I see [this](https://cs.opensource.google/go/x/sys/+/master:cpu/cpu_arm64.go;l=16-39) in go, but I don't see all of the flags you're mentioning here.

Reply:
https://github.com/torvalds/linux/blob/master/arch/arm64/include/uapi/asm/hwcap.h
https://github.com/torvalds/linux/blob/master/arch/powerpc/include/uapi/asm/cputable.h
http://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/unix/sysv/linux/s390/bits/hwcap.h;hb=HEAD

Reply:
I think for future reference, https://github.com/torvalds/linux/blob/master/arch/s390/include/asm/elf.h is a better source for cpu flags for s390x. Though in the end glibc is probably getting it from there anyway.